src/components/i18n/DopplerIntlProvider.test.tsx

```diff
@@ -1,33 +1,97 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
 import { DopplerIntlProvider } from "./DopplerIntlProvider";
-import { render } from "@testing-library/react";
-import "@testing-library/jest-dom/extend-expect";
 import { FormattedMessage } from "react-intl";
+import { useAppSessionState } from "../AppSessionStateContext";
+import { AppSessionState } from "../../abstractions/app-session";
 
-describe("DopplerIntlProvider", () => {
-  it("should render Campaign in English", () => {
-    const { getByText } = render(
-      <DopplerIntlProvider locale="en">
-        <FormattedMessage id="campaign" />
-      </DopplerIntlProvider>
-    );
-    getByText("Campaign");
-  });
+jest.mock("../AppSessionStateContext");
+jest.mock("./en", () => ({
+  messages_en: {
+    lang: "en",
+  },
+}));
+jest.mock("./es", () => ({
+  messages_es: {
+    lang: "es",
+  },
+}));
 
-  it("should render Campaign in Spanish", () => {
-    const { getByText } = render(
-      <DopplerIntlProvider locale="es">
-        <FormattedMessage id="campaign" />
-      </DopplerIntlProvider>
+const UNKNOWN_SESSION: AppSessionState = {
+  status: "unknown",
+};
+const NON_AUTHENTICATED: AppSessionState = {
+  status: "non-authenticated",
+};
+const AUTHENTICATED: AppSessionState = {
+  lang: "en",
+  status: "authenticated",
+  jwtToken: "jwt",
+  unlayerUser: {
+    id: "000",
+    signature: "000",
+  },
+  dopplerAccountName: "doppler_mock@mail.com",
+};
```

Comment:
If I am not wrong, you only need a SimplifiedAppSessionState.

```suggestion
const AUTHENTICATED: SimplifiedAppSessionState = {
  lang: "en",
  status: "authenticated",
  unlayerUser: {
    id: "000",
    signature: "000",
  },
  dopplerAccountName: "doppler_mock@mail.com",
};
```